pkg/virtctl/imageupload/imageupload.go

```diff
@@ -445,7 +445,7 @@ func waitDvUploadScheduled(client kubecli.KubevirtClient, namespace, name string
 			return false, err
 		}
 
-		if dv.Status.Phase == cdiv1.WaitForFirstConsumer {
+		if dv.Status.Phase == cdiv1.WaitForFirstConsumer && !forceBind {
```

Comment:
That is good catch, it was an actual error so it is a bug-fix, not a test fix.

Reply:
I found this issue when adapted the tests to consider also Filesystem volumes. Was not sure if the flag was ignored there intentionally or not. Thanks for having a look.